packages/fxa-payments-server/src/routes/Subscriptions/SubscriptionItem.tsx

```diff
@@ -254,94 +254,4 @@ const CancelSubscriptionPanel = ({
   );
 };
 
-type ReactivateSubscriptionPanelProps = {
```

Comment:
A lot of this PR is refactoring this bit out into a couple of smaller sub-components.